tests/behat/features/web/mediapackage.feature

```diff
@@ -56,3 +46,21 @@ Feature:
     And I should see media file with id 7 in category "Luna & Cat Theme Special"
     And I should see 1 media file in category "Luna & Cat Theme Special"
     And I should see 1 media file in category "Bla"
+
+  Scenario: Selecting and deselecting media files for downloading
+    Given I am on "/app/media-library/looks"
+    And I click "#mediafile-1"
+    And I click "#mediafile-5"
```

Comment:
is something triggered by those clicks for which we may need to wait? Ajax or something else?

Reply:
The clicks mark certain mediafiles as "selected" which means they get a border around it.